server/seed/demoSeed/generateItems.js

```diff
@@ -9,31 +9,60 @@ const generateItems = (demoUserId, userIds) => [
     _id: ObjectId(),
     authorId: demoUserId,
     description: '',
+    editedBy: ObjectId(),
```

Comment:
I'm not sure, but I think this shouldn't be fake id. These items will be displayed to the demo user and controller will try to populate this data from users collection. Replace with one of demo users id like in authorId's case eg. `[userIds[2]]`